xgraphics/new.go

```diff
@@ -261,6 +261,13 @@ func readDrawableData(X *xgbutil.XUtil, ximg *Image, did xproto.Drawable,
 					A: 0xff,
 				}
 			})
+		default:
+			return fmt.Errorf("The image returned for pixmap id %d has "+
```

Comment:
Could you, please, differentiate this error message from the one on line L284?
Or make them both the same as in depth 1 (`fmt.Errorf("The image returned for pixmap id %d with depth %d has an unsupported value for bits-per-pixel: %d",did, format.Depth, format.BitsPerPixel)`)

Without this you wouldn't know from the logs which branch of the switch for depth is the case (24 or 32), if an error happens.

Reply:
Done. I have made messages consistent and tested again.